pytorch_lightning/callbacks/base.py

```diff
@@ -101,7 +101,12 @@ def on_train_epoch_start(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningMo
     def on_train_epoch_end(
         self, trainer: 'pl.Trainer', pl_module: 'pl.LightningModule', unused: Optional = None
     ) -> None:
-        """Called when the train epoch ends."""
+        """Called when the train epoch ends.
+
+        To access all batch outputs at the end of the epoch, either:
+        1. Implement `training_epoch_end` in the `LightningModule` and access outputs via the module OR
```

Comment:
```suggestion
        To access all batch outputs at the end of the epoch, either:

        1. Implement `training_epoch_end` in the `LightningModule` and access outputs via the module OR
```
if you want a numbered list, otherwise it's also ok like this.